### PR TITLE
Added XRayBinaryClassifier model

### PR DESCRIPTION
**Pull request by akhilsinghal1234**
Unit testing to be done

### CODE REVIEW

**Comment by swapsha96** [comment]
I see a `fit` function is missing and `optimizer` is not declared. I will suggest 

1. Adding `fit function that takes `X` and `y` as input and returning output.
2. Taking `optimizer` as an input in `__init__` with default value as `None`. In `fit` function if `self.optimizer` is `None`, `Adam` optimizer is used for the same.

**Comment by swapsha96** [comment]
Also,
1. Rename `classifier.py` to `classifiers.py`. Do the same with `model.py` and rename it to `models.py`.

**Comment by swapsha96** [comment]
Could you put an example code of this class here? We'll put it in the docs later.

**Comment by akhilsinghal1234** [comment]
Instead of adding optimizer and activation in init, shouldn't they be in build model?

**Comment by swapsha96** [comment]
Oh yes. You are right. I guess then it's better to call that function `compile` rather than `build model`. (I saw another example where they used that name).

**Comment by swapsha96** [comment]
Also, looks at the comments by @pep8speaks above. It refreshes after every commit. Very useful!

**Comment by swapsha96** [comment]
One small change is left (see above). Also, since the dataset is small, data the files in a folder in `data` module along with a loader function.

**Comment by akhilsinghal1234** [comment]
Please comment any changes needed. Added train.py and utils.py too.

**Comment by swapsha96** [comment]
Please change the location of dataset from `mirapy/dataset/XRayBinary/Dataset/` to `mirapy/dataset/XRayBinary/` and set is as the default value of `data_directory` in `load_dataset`.

Also, I have added `load_dataset.py` in `data` module. Remind me to move `load_dataset` function in `utils.py` to that script (maybe calling the function `load_XRB_binary_dataset`.

Also, resolve add PEP8 issues mentioned above by @pep8speaks.

File: mirapy/classifiers/utils.py [hunk 1]
```diff
@@ -43,16 +43,16 @@ def load_data(data_directory, test_split):
...
 
-    X = df.drop('class', axis=1).values
+    x = df.drop('class', axis=1).values
     y = df['class'].values
 
     x_train, x_test, y_train, y_test = \
-        train_test_split(X, y, test_size=test_split, random_state=42)
+        train_test_split(x, y, test_size=test_split, random_state=42)
 
     y_cat = to_categorical(y_train)
```
Comment (swapsha96) [inline mirapy/classifiers/utils.py]: Do the same for `y_test`.

File: mirapy/classifiers/utils.py [hunk 2]
```diff
@@ -5,10 +5,10 @@
 from keras.utils.np_utils import to_categorical
 
 
-def load_data(data_directory, test_split):
+def load_data(data_directory, test_split, standard_scaler):
```
Comment (swapsha96) [inline mirapy/classifiers/utils.py:8]: Set the default value of `standard_scalar` to `True`.

**Comment by swapsha96** [comment]
@akhilsinghal1234 remove `1.h5` file and resolve `to_categorical` issue.

File: mirapy/classifiers/train.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+def train(model):
+    if not type(x_train) == np.ndarray and type(y_train) == np.ndarray:
+        raise ValueError('Input array should be numpy arrays')
+    history = model.fit(x_train, y_train, epochs=args.num_epochs,
+                        shuffle=True, batch_size=args.batch_size,
+                        validation_split=args.validation_split)
+
+
+if __name__ == '__main__':
```
Comment (swapsha96) [inline mirapy/classifiers/train.py:52]: Remove this part. **We never code this one in Python packages!!!**

File: mirapy/classifiers/models.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import os
+from keras.models import load_model, Model
+from keras.layers import Input, Dense
+import warnings
+warnings.filterwarnings('ignore')
+
+
+class Classifier:
```
Comment (swapsha96) [inline mirapy/classifiers/models.py:8]: Add compile abstract method based on the below class.

**Review by swapsha96 — APPROVED** [review]

**Comment by swapsha96** [comment]
LGTM!

**Pull request merged**